### ISSUE TITLE
A couple new features and minor changes

### ISSUE THREAD

**Issue by TDLemon8** [bertoldia/files-menu#6]
Hey @bertoldia,
I found and downloaded your extension and I found it great! It just didn't exactly do what I specifically wanted, so I tried to change something to make it exactly suit my needs.
Since I'm forced to be at home and I'm bored, I actually spent quite a bit of time working on it for the last week and I changed some things that I find interesting - so I thought about contacting you to see if you're as interested as I am.
Now, I never wrote an extension for GnomeShell before, and my experience with object-oriented programming is extremely limited, so I basically spent some time reading and stealing pieces of code from various tutorials. Please forgive me if I wrote something strange.
This is also my very first interaction with Github, so please tell me if I'm not contacting you in the right way.
Thank you for your time and greetings from Austria
Tullio

**Comment by TDLemon8** [comment]
Indeed, I'm such a newbie that I didn't notice that the actual current version of the extension is not hosted here on Github anymore, but on GitLab - I'm very sorry.
I worked of course on the current version from GitLab, and the rest of my message still holds anyway. So, if you are interested you can contact me here or on GitLab (same username).
Sorry again...

**Comment by bertoldia** [comment]
Hey Tullio,

Cool! Good to hear the covid pandemic self isolation thing has some up sides. Other than changes required to keep the app working with new releases, I don't have plans to add anything to the applet _myself_. Having said that, contributions are totally welcomed.

The thing to do now is for you to submit a Pull Request (or Merge Request in Gitlab parlance) with your changes. Then I'll look them over, and we can go from there.

Sound OK? Do you know how to create an MR? Here's Gitlab's instructions
https://docs.gitlab.com/ee/user/project/merge_requests/creating_merge_requests.html

Look forward to seeing your changes.
Cheers,
axel.

**Comment by TDLemon8** [comment]
Hey Axel,
thank you for your reply and your help! I had a look at the instructions
and created the MR.
I've also tried to put together a small changelog... I've fiddled with the
code so much that I didn't remember what I changed anymore, so I hope I
didn't forget anything.
Greetings
Tullio
